File: backend/src/main/java/store/andefi/core/enums/OrderStatus.java
package store.andefi.core.enums;

public enum OrderStatus {
    UNPAID("UNPAID"),
    PAID("PAID"),
    PACKED("PACKED"),
    SHIPPED("SHIPPED"),
    COMPLETED("COMPLETED"),
    CANCELED("CANCELED"),
    REFUNDED("REFUNDED"),
    PARTIAL_REFUNDED("PARTIAL_REFUNDED");

    private final String value;

    OrderStatus(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return this.value;
    }
}

File: frontend/app/src/main/java/store/andefi/utility/OrderStatus.kt
package store.andefi.utility

enum class OrderStatus(private val value: String) {
    UNPAID("unpaid"),
    PAID("paid"),
    PACKED("packed"),
    SHIPPED("shipped"),
    COMPLETED("completed"),
    CANCELED("canceled");

    override fun toString(): String {
        return this.value
    }
}
